### PR TITLE
Quality: PowerShell Summit GH-3344

### LINKED ISSUE #3344: Quality: WinSvr2022 Set-AdUser
### Prerequisites

- [X] **Existing Issue:**

Search the existing [quality issues][b1] for this repository. If there's an issue that
covers the quality area and articles you want to improve, pick a different quality area
or article set. Don't file a new issue for the same work. Subscribe, react, or comment
on that issue instead.

[b1]: https://github.com/orgs/MicrosoftDocs/projects/15/views/4

- [X] **Descriptive Title:**

Write the title for this issue as a short synopsis. If possible, provide context. For
example, "Quality: Command Syntax in Foo" instead of "Quality: Foo"

### Quality Areas

Formatting code samples, Formatting command syntax

### Article List

Set-Aduser

### PR DESCRIPTION
**Pull request by thepowerstring**
Updated set-aduser help

# PR Summary

Reflowed text to comply with 100 character limit
Added whitespace after headings
Removed trailing whitespace
Updated formatting to comply with standards

- Fixes #3344

## PR Checklist

- [X] **Descriptive Title:** This PR's title is a synopsis of the changes it proposes.
- [X] **Summary:** This PR's summary describes the scope and intent of the change.
- [X] **Contributor's Guide:** I have read the [contributors guide][contrib].
- [X] **Style:** This PR adheres to the [style guide][style].

[contrib]: https://learn.microsoft.com/powershell/scripting/community/contributing/overview
[style]: https://learn.microsoft.com/powershell/scripting/community/contributing/powershell-style-guide

### CODE REVIEW

**Comment by thepowerstring** [comment]
@microsoft-github-policy-service agree

File: docset/winserver2022-ps/activedirectory/Set-ADUser.md [hunk 1]
```diff
@@ -781,19 +830,21 @@ Accept wildcard characters: False
...
 Specifies an Active Directory user object by providing one of the following property values.
 The identifier in parentheses is the LDAP display name for the attribute.
 The acceptable values for this parameter are:
 
 - A distinguished name
-- A GUID (objectGUID) 
-- A security identifier (objectSid) 
+- A GUID (objectGUID)
+- A security identifier (objectSid)
 - A SAM account name (sAMAccountName)
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/activedirectory/Set-ADUser.md:841]:
```suggestion
- A GUID (**objectGUID**)
- A security identifier (**objectSid**)
- A SAM account name (**sAMAccountName**)
```

File: docset/winserver2022-ps/activedirectory/Set-ADUser.md [hunk 2]
```diff
@@ -826,12 +878,15 @@ Accept wildcard characters: False
...
-The **ADUser** object specified as the value of the *Instance* parameter must have been retrieved by using the **Get-ADUser** cmdlet.
-When you specify the *Instance* parameter, you cannot specify other parameters that set individual properties on the object.
+Specifies an **ADUser** object that identifies the Active Directory user object that should be
+modified and the set of changes that should be made to that object. When this parameter is
+specified, any modifications made to the **ADUser** object are also made to the corresponding Active
+Directory object. The cmdlet only updates the object properties that have changed.
+
+The **ADUser** object specified as the value of the _Instance_ parameter must have been retrieved by
+using the **Get-ADUser** cmdlet. When you specify the _Instance_ parameter, you cannot specify other
+parameters that set individual properties on the object.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/activedirectory/Set-ADUser.md:889]:
```suggestion
The **ADUser** object specified as the value of the **Instance** parameter must have been retrieved
by using the `Get-ADUser` cmdlet. When you specify the **Instance** parameter, you cannot specify
other parameters that set individual properties on the object.
```

File: docset/winserver2022-ps/activedirectory/Set-ADUser.md [hunk 3]
```diff
@@ -846,22 +901,27 @@ Accept wildcard characters: False
...
 
 - None
 - DES
 - RC4
 - AES128
 - AES256
 
-None removes all encryption types from the account, resulting in the KDC being unable to issue service tickets for services using the account.
+None removes all encryption types from the account, resulting in the KDC being unable to issue
+service tickets for services using the account.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/activedirectory/Set-ADUser.md:916]:
```suggestion
`None` removes all encryption types from the account, resulting in the KDC being unable to issue
service tickets for services using the account.
```

File: docset/winserver2022-ps/activedirectory/Set-ADUser.md [hunk 4]
```diff
@@ -846,22 +901,27 @@ Accept wildcard characters: False
...
-DES is a weak encryption type that is not supported by default since Windows 7 and Windows Server 2008 R2.
+DES is a weak encryption type that is not supported by default since Windows 7 and Windows Server
+2008 R2.
 
-Warning: Domain-joined Windows systems and services such as clustering manage their own **msDS-SupportedEncryptionTypes** attribute.
-Therefore any changes to the flag on the **msDS-SupportedEncryptionTypes** attribute are overwritten by the service or system that manages the setting.
+Warning: Domain-joined Windows systems and services such as clustering manage their own
+**msDS-SupportedEncryptionTypes** attribute. Therefore any changes to the flag on the
+**msDS-SupportedEncryptionTypes** attribute are overwritten by the service or system that manages
+the setting.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/activedirectory/Set-ADUser.md:924]:
```suggestion
> [!WARNING]
> Domain-joined Windows systems and services such as clustering manage their own
> **msDS-SupportedEncryptionTypes** attribute. Therefore any changes to the flag on the
> **msDS-SupportedEncryptionTypes** attribute are overwritten by the service or system that manages
> the setting.
```

File: docset/winserver2022-ps/activedirectory/Set-ADUser.md [hunk 5]
```diff
@@ -877,10 +937,11 @@ Accept wildcard characters: False
...
 ### -LogonWorkstations
-Specifies the computers that the user can access.
-To specify more than one computer, create a single comma-separated list.
-You can identify a computer by using the Security Account Manager (SAM) account name (sAMAccountName) or the DNS host name of the computer.
-The SAM account name is the same as the NetBIOS name of the computer.
+
+Specifies the computers that the user can access. To specify more than one computer, create a single
+comma-separated list. You can identify a computer by using the Security Account Manager (SAM)
+account name (sAMAccountName) or the DNS host name of the computer. The SAM account name is the same
+as the NetBIOS name of the computer.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/activedirectory/Set-ADUser.md:944]:
```suggestion
Specifies the computers that the user can access. To specify more than one computer, create a
single comma-separated list. You can identify a computer by using the Security Account Manager
(SAM) account name (**sAMAccountName**) or the DNS host name of the computer. The SAM account name
is the same as the NetBIOS name of the computer.
```

File: docset/winserver2022-ps/activedirectory/Set-ADUser.md [hunk 6]
```diff
@@ -897,15 +958,16 @@ Accept wildcard characters: False
...
 This parameter is set by providing one of the following property values.
 Note: The identifier in parentheses is the LDAP display name for the property.
 The acceptable values for this parameter are:
 
 - A distinguished name
-- A GUID (objectGUID) 
-- A security identifier (objectSid) 
+- A GUID (objectGUID)
+- A security identifier (objectSid)
 - A SAM account name (sAMAccountName)
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/activedirectory/Set-ADUser.md:971]:
```suggestion
- A GUID (**objectGUID**)
- A security identifier (**objectSid**)
- A SAM account name (**sAMAccountName**)
```

File: docset/winserver2022-ps/activedirectory/Set-ADUser.md [hunk 7]
```diff
@@ -1008,27 +1075,35 @@ Accept wildcard characters: False
 ```
 
 ### -Partition
+
 Specifies the distinguished name of an Active Directory partition.
 The distinguished name must be one of the naming contexts on the current directory server.
-The cmdlet searches this partition to find the object defined by the *Identity* parameter.
+The cmdlet searches this partition to find the object defined by the _Identity_ parameter.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/activedirectory/Set-ADUser.md:1081]:
```suggestion
The cmdlet searches this partition to find the object defined by the **Identity** parameter.
```

File: docset/winserver2022-ps/activedirectory/Set-ADUser.md [hunk 8]
```diff
@@ -1008,27 +1075,35 @@ Accept wildcard characters: False
...
 
-In many cases, a default value is used for the *Partition* parameter if no value is specified.
-The rules for determining the default value are given below.
-Note that rules listed first are evaluated first and when a default value can be determined, no further rules are evaluated.
+In many cases, a default value is used for the _Partition_ parameter if no value is specified. The
+rules for determining the default value are given below. Note that rules listed first are evaluated
+first and when a default value can be determined, no further rules are evaluated.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/activedirectory/Set-ADUser.md:1085]:
```suggestion
In many cases, a default value is used for the **Partition** parameter if no value is specified.
The rules for determining the default value are given below. Note that rules listed first are
evaluated first and when a default value can be determined, no further rules are evaluated.
```

File: docset/winserver2022-ps/activedirectory/Set-ADUser.md [hunk 9]
```diff
@@ -1008,27 +1075,35 @@ Accept wildcard characters: False
...
 
-In AD DS environments, a default value for *Partition* are set in the following cases: 
+In AD DS environments, a default value for _Partition_ are set in the following cases:
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/activedirectory/Set-ADUser.md:1087]:
```suggestion
In AD DS environments, a default value for **Partition** are set in the following cases:
```

File: docset/winserver2022-ps/activedirectory/Set-ADUser.md [hunk 10]
```diff
@@ -1008,27 +1075,35 @@ Accept wildcard characters: False
...
 
-- If the *Identity* parameter is set to a distinguished name, the default value of *Partition* is automatically generated from this distinguished name.
-- If running cmdlets from an Active Directory provider drive, the default value of *Partition* is automatically generated from the current path in the drive. 
-- If none of the previous cases apply, the default value of *Partition* is set to the default partition or naming context of the target domain.
+- If the _Identity_ parameter is set to a distinguished name, the default value of _Partition_ is
+  automatically generated from this distinguished name.
+- If running cmdlets from an Active Directory provider drive, the default value of _Partition_ is
+  automatically generated from the current path in the drive.
+- If none of the previous cases apply, the default value of_Partition_ is set to the default
+  partition or naming context of the target domain.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/activedirectory/Set-ADUser.md:1094]:
```suggestion
- If the **Identity** parameter is set to a distinguished name, the default value of **Partition**
  is automatically generated from this distinguished name.
- If running cmdlets from an Active Directory provider drive, the default value of **Partition** is
  automatically generated from the current path in the drive.
- If none of the previous cases apply, the default value of **Partition** is set to the default
  partition or naming context of the target domain.
```

File: docset/winserver2022-ps/activedirectory/Set-ADUser.md [hunk 11]
```diff
@@ -1008,27 +1075,35 @@ Accept wildcard characters: False
...
 
-In AD LDS environments, a default value for *Partition* will be set in the following cases:
+In AD LDS environments, a default value for _Partition_ will be set in the following cases:
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/activedirectory/Set-ADUser.md:1096]:
```suggestion
In AD LDS environments, a default value for **Partition** will be set in the following cases:
```

File: docset/winserver2022-ps/activedirectory/Set-ADUser.md [hunk 12]
```diff
@@ -1008,27 +1075,35 @@ Accept wildcard characters: False
...
-- If none of the previous cases apply, the *Partition* parameter does not take any default value.
+- If the _Identity_ parameter is set to a distinguished name, the default value of _Partition_ is
+  automatically generated from this distinguished name.
+- If running cmdlets from an Active Directory provider drive, the default value of _Partition_ is
+  automatically generated from the current path in the drive.
+- If the target AD LDS instance has a default naming context, the default value of _Partition_ is
+  set to the default naming context. To specify a default naming context for an AD LDS environment,
+  set the **msDS-defaultNamingContext** property of the Active Directory directory service agent
+  object (**nTDSDSA**) for the AD LDS instance.
+- If none of the previous cases apply, the _Partition_ parameter does not take any default value.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/activedirectory/Set-ADUser.md:1106]:
```suggestion
- If the **Identity** parameter is set to a distinguished name, the default value of **Partition**
  is automatically generated from this distinguished name.
- If running cmdlets from an Active Directory provider drive, the default value of **Partition** is
  automatically generated from the current path in the drive.
- If the target AD LDS instance has a default naming context, the default value of **Partition** is
  set to the default naming context. To specify a default naming context for an AD LDS environment,
  set the **msDS-defaultNamingContext** property of the Active Directory directory service agent
  object (**nTDSDSA**) for the AD LDS instance.
- If none of the previous cases apply, the **Partition** parameter does not take any default value.
```

File: docset/winserver2022-ps/activedirectory/Set-ADUser.md [hunk 13]
```diff
@@ -1059,15 +1135,17 @@ Accept wildcard characters: False
...
-This parameter also sets the **ADS_UF_DONT_EXPIRE_PASSWD** flag of the Active Directory User Account Control attribute.
-The acceptable values for this parameter are:
+
+Specifies whether the password of an account can expire. This parameter sets the
+**PasswordNeverExpires** property of an account object. This parameter also sets the
+**ADS_UF_DONT_EXPIRE_PASSWD** flag of the Active Directory User Account Control attribute. The
+acceptable values for this parameter are:
 
 - $False or 0
 - $True or 1
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/activedirectory/Set-ADUser.md:1145]:
```suggestion
- `$False` or `0`
- `$True` or `1`
```

File: docset/winserver2022-ps/activedirectory/Set-ADUser.md [hunk 14]
```diff
@@ -1082,10 +1160,11 @@ Accept wildcard characters: False
...
-This parameter also sets the **ADS_UF_PASSWD_NOTREQD** flag of the Active Directory User Account Control attribute.
-The acceptable values for this parameter are:
+
+Specifies whether the account requires a password. This parameter sets the **PasswordNotRequired**
+property of an account, such as a user or computer account. This parameter also sets the
+**ADS_UF_PASSWD_NOTREQD** flag of the Active Directory User Account Control attribute. The
+acceptable values for this parameter are:
 
 - $False or 0
 - $True or 1
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/activedirectory/Set-ADUser.md:1170]:
```suggestion
- `$False` or `0`
- `$True` or `1`
```

File: docset/winserver2022-ps/activedirectory/Set-ADUser.md [hunk 15]
```diff
@@ -1120,6 +1200,7 @@ Accept wildcard characters: False
 ```
 
 ### -PostalCode
+
 Specifies the postal code or zip code.
 This parameter sets the **PostalCode** property of a user object.
 The LDAP display name (**ldapDisplayName**) of this property is postalCode.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/activedirectory/Set-ADUser.md:1206]:
```suggestion
Specifies the postal code or zip code. This parameter sets the **PostalCode** property of a user
object. The LDAP display name (**ldapDisplayName**) of this property is `postalCode`.
```

File: docset/winserver2022-ps/activedirectory/Set-ADUser.md [hunk 16]
```diff
@@ -1171,15 +1254,18 @@ Accept wildcard characters: False
...
+property, you must use the LDAP display name. You can specify multiple values to a property by
+specifying a comma-separated list of values, and more than one property by separating them using a
+semicolon. If any of the properties have a null or empty value the cmdlet will return an error. The
+format for this parameter is:
 
 `-Remove @{Attribute1LDAPDisplayName=value1, value2, ...;   Attribute2LDAPDisplayName=value1, value2, ...; AttributeNLDAPDisplayName=value1, value2, ...}`
 
-When you use the *Add*, *Remove*, *Replace*, and *Clear* parameters together, the parameters are applied in the following sequence:
+When you use the _Add_, _Remove_, _Replace_, and _Clear_ parameters together, the parameters are
+applied in the following sequence:
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/activedirectory/Set-ADUser.md:1268]:
```suggestion
When you use the **Add**, **Remove**, **Replace**, and **Clear** parameters together, the
parameters are applied in the following sequence:
```

File: docset/winserver2022-ps/activedirectory/Set-ADUser.md [hunk 17]
```diff
@@ -1199,15 +1285,18 @@ Accept wildcard characters: False
...
+an object property, you must use the LDAP display name. You can specify multiple values to a
+property by specifying a comma-separated list of values, and more than one property by separating
+them using a semicolon. If any of the properties have a null or empty value the cmdlet will return
+an error. The format for this parameter is:
 
 `-Replace @{Attribute1LDAPDisplayName=value1, value2, ...;   Attribute2LDAPDisplayName=value1, value2, ...; AttributeNLDAPDisplayName=value1, value2, ...}`
 
-When you use the *Add*, *Remove*, *Replace*, and *Clear* parameters together, the operations will be performed in the following order:
+When you use the _Add_, _Remove_, _Replace_, and _Clear_ parameters together, the operations will be
+performed in the following order:
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/activedirectory/Set-ADUser.md:1299]:
```suggestion
When you use the **Add**, **Remove**, **Replace**, and **Clear** parameters together, the
operations will be performed in the following order:
```

File: docset/winserver2022-ps/activedirectory/Set-ADUser.md [hunk 18]
```diff
@@ -1301,7 +1396,11 @@ Accept wildcard characters: False
 ```
 
 ### -ServicePrincipalNames
-Specifies the service principal names for the account. This parameter sets the ServicePrincipalNames property of the account. The LDAP display name (ldapDisplayName) for this property is servicePrincipalName. This parameter uses the following syntax to add, remove, replace or clear service principal name values.
+
+Specifies the service principal names for the account. This parameter sets the ServicePrincipalNames
+property of the account. The LDAP display name (ldapDisplayName) for this property is
+servicePrincipalName. This parameter uses the following syntax to add, remove, replace or clear
+service principal name values.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/activedirectory/Set-ADUser.md:1403]:
```suggestion
Specifies the service principal names for the account. This parameter sets the
**ServicePrincipalNames** property of the account. The LDAP display name (**ldapDisplayName**) for
this property is `servicePrincipalName`. This parameter uses the following syntax to add, remove,
replace or clear service principal name values.
```

File: docset/winserver2022-ps/activedirectory/Set-ADUser.md [hunk 19]
```diff
@@ -1493,24 +1602,32 @@ Accept wildcard characters: False
...
 ## INPUTS
 
 ### None or Microsoft.ActiveDirectory.Management.ADUser
-A user object is received by the *Identity* parameter.
 
-A user object that was retrieved by using the **Get-ADUser** cmdlet and then modified is received by the *Instance* parameter.
+A user object is received by the _Identity_ parameter.
+
+A user object that was retrieved by using the **Get-ADUser** cmdlet and then modified is received by
+the _Instance_ parameter.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/activedirectory/Set-ADUser.md:1618]:
```suggestion
A user object is received by the **Identity** parameter.

A user object that was retrieved by using the `Get-ADUser` cmdlet and then modified is received by
the **Instance** parameter.
```

**Review by michaeltlombardi — CHANGES_REQUESTED** [review]
Thanks so much for these improvements, @thepowerstring! I made some suggestions around formatting. If you can apply those changes and review the file for other instances of formatting that need fixing, that would be great!

Here's a few things I noticed that you'll want to keep in mind, you can find more information in our [style guide][01]:

- We always use backticks around cmdlet names, variables in prose that have the `$` prefix, file extensions, file paths, and specific values (like valid values or values passed in an example).
- We always use double-asterisks to bold type names, property names, parameter names, and module names.
- We don't prefix PowerShell example code with `PS C:\>`.

[01]: https://learn.microsoft.com/powershell/scripting/community/contributing/powershell-style-guide

After you're happy with your updates, I'll take another look and we can get this merged!

**Pull request merged**